### ISSUE TITLE
horizontal scroll bar

### ISSUE THREAD

**Issue by dagolinuxoid** [javascript-tutorial/en.javascript.info#2835]
This is rather a strange behavior. In Chrome browser (Firefox doesn't affected) when you start scrolling downward horizontal scrollbar appears at the very bottom. Interestingly enough it only happens at certain view port width - in my case when the browser window is maximized (1920). The css apparently fixes the issue however I don't sure if it's a right choice to do. Can you confirm/reproduce?
```css
.page-wrapper {
  overflow-y: hidden;
}
```

**Comment by iliakan** [comment]
So you're using 1920x resolution?
PC or Mac?
Screenshot, please?

**Comment by dagolinuxoid** [comment]
My bad if it was vague 1920 x 1080  so it's actually 1080 width. This happens on Linux PC. I'm going to boot Window 11 to see if it  persist
[![hb.png](https://i.postimg.cc/7hYpd3nM/hb.png)](https://postimg.cc/BLkY11yX)

**Comment by dagolinuxoid** [comment]
This is very inconsistent. Sometimes there no horizontal scrollbar sometimes it appears (which is a little bit annoying). On Windows on full screen I have no scrollbar however shrinking the size of a window a little bit and the scrollbar appears. 
[![Screenshot-2.png](https://i.postimg.cc/pX92dvKQ/Screenshot-2.png)](https://postimg.cc/GHC0MZ79)

**Comment by iliakan** [comment]
Can't reproduce on my Mac.

**Comment by dagolinuxoid** [comment]
Yep it's inconsistent. It may or may not happen just after you scrolled pass through javascript info logo.

**Comment by ibrahim-301** [comment]
In windows, the horizontal scroll bar appears when the top `sitetoolbar` is out of view and window size is reduced.

https://user-images.githubusercontent.com/75935539/150638247-de6960f9-a1c2-4b5e-83be-8385852a5a23.mp4

It may have something to do with this empty `div` inside the `sitetoolbar` since setting its width to `0` removes the scroll bar. Adding `overflow: hidden;` to `.sitetoolbar` fixes it too:

<img width="913" alt="over" src="https://user-images.githubusercontent.com/75935539/150639196-8dfdf8c5-2bb1-4597-a6c8-e276d9a3a418.png">
